src/teleop/jetson/jetson_teleop.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+#Node for teleop-related callback functions
+
+from math import copysign
+from enum import Enum
+import rospy as ros
+from sensor_msgs.msg import Joy
+from geometry_msgs.msg import Twist
+from mrover.msg import DriveVelCmd, RAOpenLoopCmd, HandCmd
+
+
+def quadratic(val):
+    return copysign(val**2, val)
+
+def deadzone(magnitude, threshold):
+    temp_mag = abs(magnitude)
+    if temp_mag <= threshold:
+        temp_mag = 0
+    else:
+        temp_mag = (temp_mag - threshold)/(1 - threshold)
+
+    return copysign(temp_mag, magnitude)
+
+
+class Drive:
+    class LogitechAxes(Enum):
+      left_right = 0
+      forward_back = 1
+      twist = 2
+      dampen = 3
+      pan = 4
+      tilt = 5
+
+    #TODO: Reimplement Dampen Switch
+    def teleop_drive_callback(self, msg):
+
+        #teleop_twist_joy message is maxed at 0.5, multiply by 2 to make range [-1,1]
+        linear = msg.linear.x * 2
+        angular = msg.angular.z * 2
+
+        # Convert arcade drive to tank drive
+        # Inversions may need to be done for logitech joystick
+        angular_op = (angular / 2) / (abs(linear) + 0.5)
+        vel_left = linear + angular_op
+        vel_right = linear - angular_op
+
+        # Scale to be within [-1, 1], if necessary
+        if abs(vel_left) > 1 or abs(vel_right) > 1:
+            if abs(vel_left) > abs(vel_right):
+                vel_right /= abs(vel_left)
+                vel_left /= abs(vel_left)
+            else:
+                vel_left /= abs(vel_right)
+                vel_right /= abs(vel_right)
+
+        command = DriveVelCmd()
+        command.left = vel_left
+        command.right = vel_right
+
+        drive_vel_pub = ros.Publisher('/drive_vel_cmd', DriveVelCmd, queue_size=100)
+        drive_vel_pub.publish(command)
+
+class ArmControl:
+
+    #Leaving in slow mode for now for testing
+    slow_mode = True
```

Comment:
I added this slow_mode code since it was a personal preference (since i was arm operator) that I felt like adding but it's probably bad code so I'd delete it or at least rethink of whether it's necessary.